src/itsdangerous/timed.py

```diff
@@ -93,6 +93,12 @@ def unsign(self, value, max_age=None, return_timestamp=False):
                     payload=value,
                     date_signed=self.timestamp_to_datetime(timestamp),
                 )
+            if age < 0:
+                raise SignatureExpired(
+                    "Signature age %s < 0 seconds" % (age),
```

Comment:
```suggestion
                    "Signature age %s < 0 seconds" % (age,),
```

It’s not a tuple without the comma :slightly_smiling_face: